### PR TITLE
Datadog integration page

### PR DESCRIPTION
**Pull request by bgrenon**
Preview:
https://neon-next-git-bgrenon-datadog-integration-metrics-neondatabase.vercel.app/docs/guides/datadog

### CODE REVIEW

File: content/docs/guides/datadog.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+- You have a Neon account and project. If not, see [Sign up for a Neon account](/docs/get-started-with-neon/signing-up).
+- You have a Datadog account, your Datadog agent is set up, and you have the appropriate permissions to create an API key.
+
+## Steps to integrate Datadog with Neon
+
+1. In the Neon Console, navigate to the **Integrations** page in your Neon project.
+2. Locate the **Datadog** card and click **Add**.
+3. Enter your **Datadog API key**. You can generate or retrieve these from retrieve from [API Keys](https://app.datadoghq.com/organization-settings/api-keys) your Datadog organization.
+4. Select the **Datadog site** that best corresponds to your Neon project's region.
```
Comment (zaynetro) [inline content/docs/guides/datadog.md:47]: Datadog site is something that users select when creating their account. I don't think they can choose a different region on per-project basis. They will need to create a separate Datadog account if they want data to be hosted elsewhere.

image

File: content/docs/guides/datadog.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+- `project_id`
+- `endpoint_id`
+- `compute_id`
+- `job`
+
+Here's an example of the metric `db_total_size` with all labels:
+
+```json shouldWrap
+db_total_size{project_id="square-daffodil-12345678", endpoint_id="54321", compute_id="ep-shrill-blaze-b4hry7fg", job="sql-metrics"} = 10485760
```
Comment (zaynetro) [inline content/docs/guides/datadog.md:70]:
```suggestion
db_total_size{project_id="square-daffodil-12345678", endpoint_id="ep-aged-art-260862", compute_id="compute-shrill-blaze-b4hry7fg", job="sql-metrics"} 10485760
```

File: content/docs/guides/datadog.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Before getting started, ensure the following:
+
+- You have a Neon account and project. If not, see [Sign up for a Neon account](/docs/get-started-with-neon/signing-up).
+- You have a Datadog account, your Datadog agent is set up, and you have the appropriate permissions to create an API key.
+
+## Steps to integrate Datadog with Neon
+
+1. In the Neon Console, navigate to the **Integrations** page in your Neon project.
+2. Locate the **Datadog** card and click **Add**.
+3. Enter your **Datadog API key**. You can generate or retrieve these from retrieve from [API Keys](https://app.datadoghq.com/organization-settings/api-keys) your Datadog organization.
```
Comment (danieltprice) [inline content/docs/guides/datadog.md:46]:
```suggestion
3. Enter your **Datadog API key**. You can generate or retrieve [Datadog API Keys](https://app.datadoghq.com/organization-settings/api-keys) from your Datadog organization.
```

**Review by danieltprice — APPROVED** [review]

File: content/docs/guides/datadog.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+</Admonition>
+
+| Name                                     | Job                  | Description                                                                                                                                              |
+| ---------------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| connection_counts                        | sql-metrics          | Connection counts. The `state` label is reported as `state` from the `pg_stat_activity` Postgres view. Example values: `state="active"`, `state="idle"`. |
+| db_total_size                            | sql-metrics          | Size of all databases for the project, measured in bytes.                                                                                                |
+| lfc_approximate_working_set_size         | sql-metrics          | (DEPRECATED) Approximate working set size in pages of 8192 bytes.                                                                                        |
```
Comment (danieltprice) [inline content/docs/guides/datadog.md:81]: @zaynetro should we bother including a deprecated metric or is it just included in exported metrics until the metric is removed?

Comment (zaynetro) [reply content/docs/guides/datadog.md:81]: Let's not document it at least. I think we should not export it at all. I will create a task for us.

File: content/docs/guides/datadog.md (hunk 1 above)
Comment (bgrenon) [reply content/docs/guides/datadog.md:47]: Ok, got it. Looks like a roundabout way to check your region in datadog: compare your URL against a table in their docs. 
https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site

I'll revise to match this.

File: content/docs/guides/datadog.md (hunk 4 above)
Comment (bgrenon) [reply content/docs/guides/datadog.md:81]: removed.

**Pull request merged**